.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Don't merge this file.  Please remove it.

Reply:
removed .gitignore